pkg/controller/plan/kubevirt.go

```diff
@@ -893,6 +893,29 @@ func (r *KubeVirt) SetPopulatorPodOwnership(vm *plan.VMStatus) (err error) {
 	return
 }
 
+// DeletePVCs deletes PVCs associated with the VM, including prime PVCs (populator flows)
+func (r *KubeVirt) DeletePVCs(vm *plan.VMStatus) (err error) {
```

Comment:
could be nicer to move it into destination client

Reply:
looked into it, it will causing code duplication from the kubevirt into 2 destination clients (ovirt and openstack).